protocol/testing/snapshotting/snapshot.sh

```diff
@@ -92,7 +92,7 @@ setup_cosmovisor
 # TODO: add metrics around snapshot upload latency/frequency/success rate
 while true; do
   # p2p.seeds taken from --p2p.persistent_peers flag of full node
-  cosmovisor run start --log_level info --home /dydxprotocol/chain/local_node --p2p.seeds "${p2p_seeds}" --non-validating-full-node=true &
+  cosmovisor run start --log_level info --home /dydxprotocol/chain/local_node --p2p.seeds "${p2p_seeds}" --bridge-daemon-eth-rpc-endpoint "https://eth-sepolia.g.alchemy.com/v2/demo" --non-validating-full-node=true &
```

Comment:
possible to multi-line this command using `\` to extend lines? (for readability)